doc/make_external_gallery.py

```diff
@@ -150,6 +150,24 @@ def format(self):
         link="https://github.com/JacobBumgarner/grad-descent-visualizer",
         image="grad_descent_visualizer.gif",
     ),
+    nikolov1=Example(
+        title="Python Libraries for Mesh, Point Cloud, and Data Visualization (Part 1)",
+        description="Python Libraries for Mesh, Point Cloud, and Data Visualization (Part 1)",
+        link="https://medium.com/towards-data-science/python-libraries-for-mesh-and-point-cloud-visualization-part-1-daa2af36de30",
+        image="nikolov1.gif",
+    ),
+    nikolov2=Example(
+        title="How to Voxelize Meshes and Point Clouds in Python",
+        description="How to Voxelize Meshes and Point Clouds in Python",
+        link="https://medium.com/towards-data-science/how-to-voxelize-meshes-and-point-clouds-in-python-ca94d403f81d",
+        image="nikolov2.gif",
+    ),
+    nikolov3=Example(
+        title="Neighborhood Analysis, KD-Trees, and Octrees for Meshes and Point Clouds in Python",
+        description="Neighborhood Analysis, KD-Trees, and Octrees for Meshes and Point Clouds in Python",
+        link="https://medium.com/towards-data-science/neighborhood-analysis-kd-trees-and-octrees-for-meshes-and-point-clouds-in-python-19fa96527b77",
+        image="nikolov3.gif",
```

Comment:
One more, sorry: it seems to me that this nice yellow bunny gif was made with Vedo, not PyVista :( We should double-check that all 3 gifs were made with PyVista (the first two are probably OK, but I only skimmed the three posts a bit).

Reply:
Thank you for the check. Sorry for the mistake. This is due to the fact that the word "PyVista" was on the right and "Vedo" was on the left. I should have looked at the captions more carefully.
![image](https://user-images.githubusercontent.com/7513610/200087834-1f261d65-827b-40ad-8332-f208a0776577.png)

Reply:
Fixed in a1b7aee0a142558d73069415e02d3467a3abcc5f .